modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/PersistentPageMemoryStorageEngine.java

```diff
@@ -159,11 +165,11 @@ public void stop() throws StorageException {
             Stream<AutoCloseable> closeRegions = regions.values().stream().map(region -> region::stop);
 
             CheckpointManager checkpointManager = this.checkpointManager;
-            PartitionFilePageStoreManager partitionFilePageStoreManager = this.partitionFilePageStoreManager;
+            FilePageStoreManager filePageStoreManager = this.filePageStoreManager;
 
             Stream<AutoCloseable> closeManagers = Stream.of(
                     checkpointManager == null ? null : (AutoCloseable) checkpointManager::stop,
-                    partitionFilePageStoreManager == null ? null : (AutoCloseable) partitionFilePageStoreManager::stop
+                    filePageStoreManager == null ? null : (AutoCloseable) filePageStoreManager::stop
```

Comment:
All these null checks won't be required anymore

Reply:
Is it so? Can't it be that the **stop** method will be called before the **start** method?

Reply:
"stop" can't be called before constructor

Reply:
Then why can't there be null's here?